ccic/tiler.py

```diff
@@ -228,11 +263,10 @@ def assemble(self, slices):
                 row_slice = slice(i_start, i_end)
                 j_start = self.j_start[j]
                 j_end = j_start + self.tile_size[1]
-                col_slice = slice(j_start, j_end)
+                # modulo self.n in case self.wrap_columns is True
+                col_slice = np.arange(j_start, j_end) % self.n
 
-                output = results[..., row_slice, col_slice]
-                weights = self.get_weights(i, j)
-                output += weights * slc
+                results[..., row_slice, col_slice] += self.get_weights(i, j)*slc
```

Comment:
I didn't find a simpler way to make it consistent with the previous code, so I replaced it with this

Reply:
Looks good.